tests/scripts/t3000/run_t3000_frequent_tests.sh

```diff
@@ -64,7 +64,13 @@ run_t3000_mixtral_tests() {
   echo "LOG_METAL: Running run_t3000_mixtral_tests"
 
   # mixtral8x7b 8 chip decode model test (env flags set inside the test)
+  # Decode-only
   pytest models/demos/t3000/mixtral8x7b/tests/test_mixtral_model.py::test_mixtral_model_inference[wormhole_b0-True-10-1-pcc]
+  # Prefill-only (32L)
+  pytest models/demos/t3000/mixtral8x7b/tests/test_mixtral_model_prefill.py::test_mixtral_model_inference[wormhole_b0-True-32-128]
```

Comment:
can these tests be run using pytest parameters instead of invoking multiple times?